.github/workflows/nightly_build.yaml

```diff
@@ -14,11 +14,19 @@ jobs:
 
     permissions:
       contents: read
+      id-token: write
       packages: write
 
+    env:
+      COSIGN_EXPERIMENTAL: 1
+
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      - name: Install cosign
```

Comment:
so, we'll be signing only release and nightly builds, how about regular builds images?  (the ones generated when running PRs) 
I think signing regular PR images can be useful to verify sign still working, and we may benefits with that if we create an IT that verify our images are signed

Reply:
agree, you can sign the image that you generate in the main branch build

Reply:
For that there is already a nightly in place. On the main branch no images are released, so also nothing to sign there.